modules/appeals_api/app/workers/appeals_api/higher_level_review_pdf_submit_job.rb

```diff
@@ -75,5 +74,27 @@ def receive_date(higher_level_review)
         .in_time_zone('Central Time (US & Canada)')
         .strftime('%Y-%m-%d %H:%M:%S')
     end
+
+    def log_upload_error(higher_level_review, e)
+      Rails.logger.info("#{higher_level_review.class.to_s.gsub('::', ' ')}: Submission failure",
+                        'source' => higher_level_review.consumer_name,
+                        'consumer_id' => higher_level_review.consumer_id,
+                        'consumer_username' => higher_level_review.consumer_name,
+                        'uuid' => higher_level_review.id,
+                        'code' => e.code,
+                        'detail' => e.detail)
+    end
+
+    def handle_upload_error(higher_level_review, e)
+      log_upload_error(higher_level_review, e)
+      higher_level_review.update(status: 'error', code: e.code, detail: e.detail)
+
+      if e.code == 'DOC201' || e.code == 'DOC202'
```

Comment:
I feel like we should notify in this case as well -feels kinda bad form to wait for our daily error report to know that this stalled out and needs manual intervention.

Reply:
good point